doc/select.md

```diff
@@ -0,0 +1,84 @@
+# Select examples
```

Comment:
The main idea of these examples is to show `crud.select` capabilities, but also to teach user how to use it.

I think that select examples should be reordered:

* Filters - show using conditions by index, by composite index, by first part of index, by non-indexed field (add a note that in this case full-scan is performed). We also need to show that condition by indexed field should be placed first to avoid full-scan.

* Pagination - first, after, first and after together, reverse pagination.

Maybe we can show the simple loop that scans space and shows results by pages, or add a comments to your examples, smth like "Ok, we've got first 3 tuples, let's get next 3 tuples - specify row[3] as an `after`".